### PR TITLE
DHFPROD-2135: FE Implementation Display MLCP command in ingest step

### PR DESCRIPTION
**Pull request by xnikhil08**
1. Added the MLCP Command in the ingest step.
2. It can be pasted directly onto the terminal, after replacing the password parameter with actual values. Currently set as an encrypted value "*****".
MLCPCommand
@wooldridge - Please review it.

### CODE REVIEW

File: web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts [hunk 1]
```diff
@@ -180,7 +174,6 @@ export class EditFlowComponent implements OnInit, OnDestroy {
   }
   updateStep(step) {
     this.manageFlowsService.updateStep(this.flow.id, step.id, step).subscribe(resp => {
-      this.editFlowUi.stepUpdated(step);
```
Comment (brucean52) [inline web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts]: It looks like some functionality that's on develop has been removed with this PR. Please look into this

File: web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts [hunk 2]
```diff
@@ -55,8 +54,6 @@ export class EditFlowComponent implements OnInit, OnDestroy {
   projectDirectory: string;
   flowEnded: string = '';
   stepType: typeof StepType = StepType;
-  runFlowClicked: boolean = false;
-  disableSelect: boolean = false;
```
Comment (brucean52) [inline web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts]: It looks like some functionality that's on develop has been removed with this PR. Please look into this

**Comment by srinathgit** [comment]
The step number should also be included, else, it will always run the first step

**Comment by aebadirad** [comment]
May want to rebase with develop.

File: web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts [hunk 3]
```diff
@@ -141,15 +138,14 @@ export class EditFlowComponent implements OnInit, OnDestroy {
     });
   }
   runFlow(runObject): void {
-    this.runFlowClicked = true;
     this.manageFlowsService.runFlow(runObject).subscribe(resp => {
-      // TODO add error handling for error response, set runFlowClicked = false;
+      // TODO add response check
+      console.log('run flow resp', resp);
```
Comment (rjrudin) [inline web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts]: Not sure of the team standards yet, but I usually vote on removing log statements like this

**Pull request closed without merging**